generator/srtm_coverage_checker/srtm_coverage_checker.cpp

```diff
@@ -75,8 +57,9 @@ int main(int argc, char * argv[])
   platform::FindAllLocalMapsAndCleanup(numeric_limits<int64_t>::max() /* latestVersion */,
                                        localFiles);
 
-  auto fetcher = CreateFeaturesFetcher(localFiles);
   generator::SrtmTileManager manager(FLAGS_srtm_path);
+  classificator::Load();
+  classif().SortClassificator();
```

Comment:
Why do we need this? (SortClassificator)?

Reply:
No. Not really. Removed.